libraries/redcore/database/driver.php

```diff
@@ -248,5 +247,203 @@ public static function deleteInstances()
 		{
 			self::$instances = null;
 		}
+
+		/**
+		 * Splits a string of multiple queries into an array of individual queries.
+		 *
+		 * @param   string  $sql  Input SQL string with which to split into individual queries.
+		 *
+		 * @return  array  The queries from the input string separated into an array.
+		 *
+		 * @since   11.1
+		 */
+		public static function splitSql($sql)
```

Comment:
it won't, it's an static function https://github.com/redCOMPONENT-COM/redCORE/pull/440/files#diff-3b7de2cf163f18aa521c050bb543084fR329
it's the same reason why I had to rewrite it, because the call in Joomla is static